lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java

```diff
@@ -359,6 +363,17 @@ private void drawCharacter(char[] character, Paint paint, Canvas canvas) {
     return contents;
   }
 
+  private String codePointToString(int codePoint) {
```

Comment:
move to a utility? Doesn't seem specific to `TextLayer`

Reply:
@digitalbuddha It's not used anywhere else though.

Reply:
fair enough, I know it currently does not have a test around it. If you were to write a test would you then create an impl of TextLayer to do it? Its a nitpick and your choice

Reply:
I just use the snapshot tests as a test. We could do unit tests too but they would be redundant with the snapshots.